src/utils/utils.ts

```diff
@@ -87,3 +105,22 @@ export function getBulletedList(pretext: string, items: string[]): string {
 export function getStaticURL(context: AppContext, name:string): string {
     return context.mattermost_site_url + '/plugins/com.mattermost.apps/apps/' + getManifest().app_id + '/static/' + name;
 }
+
+export function isConfigured(context: AppContext): boolean {
+    return Boolean(context.oauth2.client_id && context.oauth2.client_secret);
+}
+
+export function isUserSystemAdmin(context: AppContext): boolean {
+    return Boolean(context.acting_user.roles && context.acting_user.roles.includes(GeneralConstants.SYSTEM_ADMIN_ROLE));
+}
+
+export function isConnected(context: AppContext): boolean {
+    if (context.oauth2.user && context.oauth2.user.access_token && context.oauth2.user.access_token !== '') {
```

Comment:
The check for empty string here is redundant

```suggestion
    if (context.oauth2.user && context.oauth2.user.access_token) {
```